source/templates/_main.slim

```diff
@@ -54,7 +54,7 @@ main[ng-controller="MainController as mainCtrl" ng-init="mainCtrl.init()"]
 		p.progress-indicator[progress-model="mainCtrl.loadAppConfigProgress"]
 		p.progress-indicator[progress-model="mainCtrl.changeMenuProgress"]
 		p.progress-indicator[progress-model="mainCtrl.initAppProgress"]
-		.tab-content[ng-view ng-if="mainCtrl.initAppProgress.isIdle && !mainCtrl.menuProgress.isError && !mainCtrl.loadAppConfigProgress.isInProgress  && mainCtrl.changeMenuProgress.isIdle"]
+		.tab-content[ng-view ng-if="mainCtrl.currentMenu.id === 'yml' || (mainCtrl.initAppProgress.isIdle && !mainCtrl.menuProgress.isError && !mainCtrl.loadAppConfigProgress.isInProgress && !mainCtrl.loadAppConfigProgress.isError  && mainCtrl.changeMenuProgress.isIdle)"]
```

Comment:
Why is this needed?

Reply:
When app is in repo mode and bitrise.yml is missing or invalid in the repo, the app had an error on top and a stuck loading indicator below. 
By adding `!mainCtrl.loadAppConfigProgress.isError` to this logic, the tab content won't show up if the appConfig has issues. However I wanted to keep the `yml` tap available to give the users an option to switch back to website mode even if the the said error is present.